website/conferences/views.py

```diff
@@ -148,6 +148,7 @@ def _render_conference_node(node, idx, conf):
             ).limit(1)
         ).wrapped()
         download_count = record.get_download_count()
+        view_and_download = download_count + node.visit
```

Comment:
`download_count` would be superfluous with these changes; it's not used anywhere other than this line, so it could be changed to `view_and_download = record.get_download_count() + node.visit`